tests/DocumentEmbedderTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Tests;
+
+use PhpLlm\LlmChain\Document\Document;
+use PhpLlm\LlmChain\Document\Vector;
+use PhpLlm\LlmChain\DocumentEmbedder;
+use PhpLlm\LlmChain\EmbeddingsModel;
+use PhpLlm\LlmChain\Store\StoreInterface;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(DocumentEmbedder::class)]
+final class DocumentEmbedderTest extends TestCase
```

Comment:
with that level of mocking i would propose to consider it a unit test and add `#[Small]` attribute here

Reply:
done, not really used to this attributes, sorry